gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/metastore/HiveMetaStoreBasedRegister.java

```diff
@@ -377,7 +378,7 @@ public Boolean call() throws Exception {
           }
         });
       } catch (ExecutionException ee) {
-        throw new IOException("Database existence checking throwing execution exception.");
+        throw new IOException("Database existence checking throwing execution exception.", ee);
```

Comment:
Do we know the typical reasons for the exception at this place, or it's hard to tell at this layer?
Ideally, exception messages should be in the format of "Problem happened because of A, do B to resolve it".

Reply:
It is hard to tell here. Mostly likely it could be Hive related issue, but maybe too vague to give users useful instructions to fix.